dotcom-rendering/src/lib/commercial-constants.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * The maximum number of fronts-banner ads that can be inserted on any front.
+ */
+export const MAX_FRONTS_BANNER_ADS = 6;
```

Comment:
Note that this applies to ads inserted via algorithm and is not applicable to pages that have a custom collection specification defined in `frontsBannerAbTestAdPositions.ts`

Reply:
Minor - it looks like this is only used by one module so it seems a little unnecessary to be extracted to its own file.

Reply:
Yep, good spot